init.lua

```diff
@@ -908,6 +908,12 @@ require('lazy').setup({
     --    - Treesitter + textobjects: https://github.com/nvim-treesitter/nvim-treesitter-textobjects
   },
 
+  { -- Review undo history with a tree view
+    'mbbill/undotree',
+    event = 'VeryLazy', -- Set the loading event to 'VeryLazy'
```

Comment:
It might be better to remove the event and load the plugin only when using it's keybinding via the [keys](https://lazy.folke.io/spec/lazy_loading#%EF%B8%8F-lazy-key-mappings) spec.